src/focus-ring.js

```diff
@@ -1,67 +1,112 @@
-/* https://github.com/WICG/focusring */
+/* https://github.com/WICG/focus-ring */
```

Comment:
oops I merged my own PR that had this same change
https://github.com/WICG/focus-ring/pull/1